baseplate/frameworks/queue_consumer/kafka.py

```diff
@@ -331,6 +331,10 @@ def make_kafka_consumer(
             # this means that when a new consumer group is created it will only
             # process new messages.
             "auto.offset.reset": "latest",
+            # Use the cooperative sticky assignor for consumer group rebalances.
+            # This *dramatically* reduces the time taken to rebalance the consumer group
+            # when the underlying topic metadata changes (leadership changes etc).
+            "partition.assignment.strategy": "cooperative-sticky",
```

Comment:
This seems like this will change how things work for existing consumers. Will it make things better in all cases?

Reply:
Once they restart their consumer applications they will use this new assignment strategy. It will only be triggered on a consumer group rebalance. If that happens they will see a much quicker rebalance as fewer partitions will be reassigned between consumers. So, yes, we should see an across the board improvement.